README.md

```diff
@@ -17,6 +17,8 @@ storage-specific optimization support.
     - `agent`: Implementation of `AbstractVolumeAgent` with XFS support
   - `purestorage`
     - PureStorage's FlashBlade-optimized backend with RapidFile Toolkit (formerly PureTools)
+  - `netapp`
+    - Reflecting the status of heterogeneous Netapp storages by facing NetApp ONTAP Rest API
```

Comment:
```suggestion
    - NetApp QTree integration backend based on the NetApp ONTAP REST API
```
* Rest API -> REST API
* Like description of all other backends, let's make it consistently a noun phrase based on "backend"

Reply:
Got it! I'll fix it right away.